ChangeLog

```diff
@@ -28,7 +28,8 @@ Bugfix
      In the context of SSL, this resulted in handshake failure. #1351
    * Fix Windows x64 builds with the included mbedTLS.sln file. #1347
    * In function mbedtls_x509_crt_parse_path resize buffer entry_name from 255 to 512,
```

Comment:
* This omits the primary reason for the change, which is to fix a build failure.
* This contains irrelevant details (especially the name of a local variable).
* There are minor English mistakes.

> In `mbedtls_x509_crt_parse_path`, check the length of the file path instead of silently truncating it. This avoids a warning from GCC 7.